src/kakarot/evm.cairo

```diff
@@ -709,9 +727,11 @@ namespace EVM {
             let (empty: felt*) = alloc();
             tempvar bytecode = calldata;
             tempvar calldata = empty;
+            tempvar intrinsic_gas = intrinsic_gas + 32000;
```

Comment:
Let's check on [revm](https://github.com/bluealloy/revm/blob/32e0edff78765eb9433b53946fc0d76bee8e08b3/crates/interpreter/src/gas/calc.rs#L340C65-L340C65) if we're changing enough, or [execution spec](https://github.com/ethereum/execution-specs/blob/master/src/ethereum/shanghai/fork.py#L689)

On second thought, we should add another issue since it's outside the scope of this PR

Reply:
I read the spec, and this is correct afaiu

Reply:
Yes it feels like it, but I keep seeing "initcode stipend" for create operations since shanghai, idk if it applies here
https://github.com/kkrt-labs/kakarot/pull/828
https://github.com/bluealloy/revm/blob/32e0edff78765eb9433b53946fc0d76bee8e08b3/crates/interpreter/src/gas/calc.rs#L379

Reply:
indeed possibly missing some parts. I just happened to (partially) fix the tx cost, but the goal of this pr is memory expansion

Reply:
Fine with me! Let's fix this in another PR